create_s3background_account_cipher.py

```diff
@@ -0,0 +1,201 @@
+    {'key' : 'users', 'func' : create_default_prepare_params},
+    {'key' : 'roles', 'func' : create_default_prepare_params},
+    {'key' : 's3userid', 'func' : create_s3userid_prepare_params},
+    {'key' : 'access_key', 'func' : create_accesskey_prepare_params},
+    {'key' : 'groups', 'func' : create_default_prepare_params},
+    {'key' : 'policies', 'func' : create_default_prepare_params}
+]
+
+def generate_key(config, use_base64, key_len, const_key):
+    s3cipher = CortxS3Cipher(config, use_base64, key_len, const_key)
+    return s3cipher.get_key()
+
+def generate_access_secret_keys(const_access_string, const_secret_string):
+    cortx_access_key = generate_key(None, True, 22, const_access_string)
+    cortx_secret_key = generate_key(None, False, 40, const_secret_string)
+    return cortx_access_key, cortx_secret_key
+
+def connect_to_ldap_server():
+    ldap_connection = ldap.initialize(LDAP_URL)
+    ldap_connection.protocol_version = ldap.VERSION3
+    ldap_connection.set_option(ldap.OPT_REFERRALS, 0)
+    ldap_connection.simple_bind_s(LDAP_USER, LDAP_PASSWORD)
+    return ldap_connection
+
+def is_account_present(account_name, ldap_connection):
+    try:
+        results = ldap_connection.search_s("o=s3-background-delete-svc,ou=accounts,dc=s3,dc=seagate,dc=com", ldap.SCOPE_SUBTREE)
+    except Exception as e:
+        return False
+
+    if len(results) < 6:
+        raise Exception("s3-background-delete-svc account creation was corrupted, use s3iamcli to DeleteAccount")
+    else:
+        return True
+
+def disconnect_from_ldap(ldap_connection):
+    ldap_connection.unbind_s()
+
+def generate_input_params():
+    input_params = g_default_input_params
+
+    bg_access_key, bg_secret_key = generate_access_secret_keys(
+        input_params['--cstr_access'],
+        input_params['--cstr_secret'])
+    
+    input_params['--secret_key'] = bg_secret_key
+    input_params['--access_key'] = bg_access_key
+
+    return input_params
+
+def create_account(input_params):
+    ldap_connection = None
+    try:
+        ldap_connection = connect_to_ldap_server()
+
+        if not is_account_present(input_params['--account_name'], ldap_connection):
+
+            for item in g_create_func_table:
+                dn, attrs = item['func'](item['key'],input_params)
+                ldif = modlist.addModlist(attrs)
+                ldap_connection.add_s(dn,ldif)
+
+            print("Account Created Successfully..")
+            print("Access Key : {}, Secret Key : {}".format(input_params['--access_key'],
+                input_params['--secret_key']))
+
+        else:
+            print("Account already present.")
+
+    except Exception as e:
+        print("Exception : {}".format(e))
+        print("Traceback : {}".format(traceback.format_exc()))
+    finally:
+        if not ldap_connection:
+            ldap_connection.unbind()
+
+
+if __name__ == "__main__":
+    input_params = generate_input_params()
+    create_account(input_params)
```

Comment:
have a print saying 'adding ..... accounts'

Reply:
Have added a print statement which lists the details of created account. Kept it in alignment with the output expected by bgdelete spec tests